Tasks/PackerBuildV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -89,9 +89,6 @@
   "loc.messages.ParsingAdditionalBuilderParameters": "Parsing additional builder parameters json.",
   "loc.messages.ParsingTemplateFileContentFailed": "Unable to parse json content from template file %s with error: %s.",
   "loc.messages.ParsingCustomTemplateParameters": "Parsing custom template parameters json.",
-  "loc.messages.FetchingSPNDetailsRemotely": "Fetching SPN details for app ID %s from the Azure AD graph service connection...",
```

Comment:
Sorry, could you please explain why these strings were removed?

Reply:
it is not releated my PR, when I run make.js build, it updated this file as well. It seems it is not relevant translation any more